### DISCUSSION TITLE
windows tutorial

### DISCUSSION THREAD

**Discussion by Biggingus** [AlexGustafsson/homebridge-wol#262] [Questions]
I would love to get this plugin working for Homebridge with my Windows PC but am a bit lost Are there any tutorials or step-by-steps I can follow 
Thank you

**Comment by flex-chillrsn** [comment]
A Windows Tutorial would be very helpful. I was able to get WoL working for my PC but can't figure out shutdown. `psshutdown -d -t 0` works from the powershell command line to sleep the PC but doesn't seem to work when used as a value in the js config.
